flask/boxwise_flask/graph_ql/query_defs.py

```diff
@@ -10,9 +10,14 @@
         base(id: Int!): Base
         allUsers: [User]
         user(email: String): User
-        box(qr_code: String): Box
+        box(id: String): Box
```

Comment:
Can we make the argument types more consistent? For box, location, qrCode, product it's a mix of String and Int. Also I remember a schema proposal from @aerinsol where they were of type ID.

Reply:
Totally agree! I would suggest to postpone these important consistency adjustments and cleanups to a several work package / PR(s) though. 
As a reminder, the backend changes here in this PR are for sake of pragmatism for enabling the FE progress.

Reply:
A good workflow in this early stage where the API / basic BE capabilities are still sketched out and FE devs are not working on a daily/weekly basis might be to ignore that BE/schema changes might break the current FE implementation. I will then follow up with the FE in the next hackathon to adapt the queries. Since FE tests are mocked, the pipeline wouldn't fail for FE and you wouldn't be blocked.

Reply:
Alright I'll take that literally ;) and try to integrate the most recent schema here already.